### PR TITLE
try-1

### PR DESCRIPTION
**Pull request by RomanSoldatov**

### CODE REVIEW

**Review by DmytroPerepelytsiaa — APPROVED** [review]

File: app.js [hunk 1]
```diff
@@ -11,6 +14,10 @@ app.use(express.json());
 app.use(express.static("public"));
 
 app.use("/api/users", authRouter);
+app.use("/api/contacts", contactsRouter);
```
Comment (DmytroPerepelytsiaa) [inline app.js:17]: we don't have contacts, delete it

File: middlewares/index.js [hunk 2]
```diff
@@ -1,11 +1,21 @@
...
-    authenticate,
-    upload,
+  validateBody,
+  isValidId,
+  contactWithoutId,
+  columnWithoutId,
+  todoWithoutId,
+  boardWithoutId,
+  authenticate,
+  isCurrentUserOwner,
```
Comment (VIPon4ik) [inline middlewares/index.js:18]: delete this

File: middlewares/index.js [hunk 3]
```diff
@@ -1,11 +1,21 @@
...
+  isFileExist,
```
Comment (VIPon4ik) [inline middlewares/index.js:19]: this too

File: routes/boardsRouter.js [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+  isCurrentUserOwner,
+  ctrl.getById
+);
+
+boardsRouter.delete(
+  "/:id",
+  isValidId,
+  boardWithoutId,
+  authenticate,
+  isCurrentUserOwner,
```
Comment (DmytroPerepelytsiaa) [inline routes/boardsRouter.js:31]: made isCurrentUserOwner for checking if user is owner of the board, and name it something like isUserOwnerOfBoard mb?

**Review by DmytroPerepelytsiaa — CHANGES_REQUESTED** [review]

File: schemas/boardsSchemas.js [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+const boardSchema = new Schema(
+  {
+    title: {
+      type: String,
+      required: true,
+      //   maxLength: 30,
+      trim: true,
+    },
+    iconsURL: String,
```
Comment (DmytroPerepelytsiaa) [inline schemas/boardsSchemas.js:12]: change it to iconURL, because you store only 1 icon

**Review by DmytroPerepelytsiaa — CHANGES_REQUESTED** [review]

**Pull request closed without merging**